renovate.json

```diff
@@ -1,5 +1,13 @@
 {
   "extends": [
-    "config:base"
-  ]
+    "config:base",
+    ":maintainLockFilesWeekly",
+    ":prNotPending",
+    ":semanticCommitsDisabled",
+    ":unpublishSafe"
+  ],
+  "reviewers": [
+    "mozilla-neutrino/core-contributors"
```

Comment:
`s/mozilla-neutrino/neutrinojs/` (though this feature doesn't yet work properly anyway, due to issues with the GitHub API)